.github/workflows/_release.yml

```diff
@@ -11,6 +11,9 @@ jobs:
         with:
           merge-multiple: true
 
+      - name: Dependencies license compliance checker
+        uses: pilosus/action-pip-license-checker@v2.0.0
```

Comment:
I think this should pin to a SHA, as this isn't a trusted source

Reply:
looking into it now

Reply:
pushed